code/datums/datum.dm

```diff
@@ -8,6 +8,7 @@
 	var/signal_enabled = FALSE
 	var/datum_flags = NONE
 	var/datum/weakref/weak_reference
+	var/list/datum_outputs

```

Comment:
How hard would it be to move this to `/atom` and just have components use the list of whatever they're playing sound out of

Reply:
they already do, we changed it to be like this so you can add squeak component to things at runtime and you wont have to fuck with their datum outputs list since it's all handled internally